ajax/libs/jodit/package.json

```diff
@@ -10,18 +10,36 @@
     "rich",
     "jodit"
   ],
-  "license": "CC-BY-NC-3.0",
+  "license": "MIT",
```

Comment:
```suggestion
  "license": "GPL-2.0-or-later",
```

The main license in the source repository appears to be GNU GPL 2.0?

Reply:
Main MIT @MattIPv4

Reply:
Ah, cool.